src/player.h

```diff
@@ -1219,6 +1233,9 @@ class Player final : public Creature, public Cylinder
 		std::forward_list<std::string> learnedInstantSpellList;
 		std::forward_list<Condition*> storedConditionList; // TODO: This variable is only temporarily used when logging in, get rid of it somehow
 
+		//quest tracker
+		std::list<TrackedQuest> trackedQuestList;
```

Comment:
Definitely use std::vector over std::list here

Reply:
I came to think of it at the time of writing it, but is there an obvious reason to make it a vector compared to a list?
the `missions`/`quests` seem to be a list because it is easy to build them with the `emplace_back` method